logsrvd/sendlog.c

```diff
@@ -932,6 +1086,9 @@ handle_server_message(uint8_t *buf, size_t len,
 
     server_message__free_unpacked(msg, NULL);
     debug_return_bool(ret);
+
+bad:
```

Comment:
The "bad" label should probably be #if defined(HAVE_OPENSSL), otherwise we get a warning from the compiler when built without OpenSSL.

Reply:
fixed!
The execution jumps to this label only when do_tls_handshake() had problems, so I removed the label and we simply return with false instead of jumping onto a label and then returning false.